odpf/assets/v1beta2/ml_feature.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package odpf.assets.v1beta2;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/odpf/proton/assets/v1beta2;assetsv1beta2";
+option java_outer_classname = "MLFeatureProto";
+option java_package = "io.odpf.assets";
+
+// MLFeature is a Machine Learning(ML) feature. In machine learning, a feature
+// is an individual measurable property, typically represented by a column,
+// that serves as an input for Machine Learning (ML) algorithms.
+message MLFeature {
+  // Feature value type. Used to define data types in Feature Tables.
+  enum DataType {
```

Comment:
@sudo-suhas do we need this enum? can this be just string?

Reply:
The idea was to homogenize the types across different vendors who might represent the same underlying type slightly differently. Don't have a strong opinion though, can just use string if you think that would be better.

Reply:
@sudo-suhas I think as metadata owners we can't enforce it anyway. So better to just use what the source system provides and keep it flexible. So my opinion would be to remove it.